test/e2e/apps/statefulset.go

```diff
@@ -86,6 +86,11 @@ var httpProbe = &v1.Probe{
 // GCE Api requirements: nodes and master need storage r/w permissions.
 var _ = SIGDescribe("StatefulSet", func() {
 	f := framework.NewDefaultFramework("statefulset")
+	f.NamespaceLabels = map[string]string{
```

Comment:
Would this be better to wrap with a function?

Reply:
@kerthcet based on the current iteration of reviews this now reduced to:
```
	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
```
My feeling is that this is small enough to not to be wrapped with a function :thinking: